irohad/main/irohad.cpp

```diff
@@ -73,6 +76,11 @@ DEFINE_string(keypair_name, "", "Specify name of .pub and .priv files");
  */
 DEFINE_validator(keypair_name, &validate_keypair_name);
 
+/**
+ * Create input argument for the initial peer list
+ */
+DEFINE_string(peers, "", "Specify initial peers");
```

Comment:
```suggestion
DEFINE_string(peers, "", "Path to file with current peers list in <...> format");
```
also I would suggest using some standard format, not just "words" (propietary format?), but rather CSV or JSON. People will feel more comfortable when they see a format they know, even if it will require them to type more symbols.

Reply:
I don't agree that JSON or CSV will be better here - in my opinion it will just make users suffer since they will need some kind of external utility for that.
But I agree that an unknown format can be uncomfortable to use so docs should be definitely updated. Does anybody know how?

Reply:
JSON and especially CSV will not make the users type much more if their initial peers lists are short. But if they are not, they will probably automate their creation (imagine merging several lists and/or some logic of peers list generation based on blockstore) - this will require them to write some code that generates the list. And in this case they will more likely prefer existing and well-known formats and some of numerous popular libraries. Also, JSON and CSV are easier to extend in the future without inventing our proprietary syntax.

Reply:
Well, that's a my position:
- Documentation is necessary for both cases anyway
- I can create a plain text file blindly but I can't do the same for JSON even for a very simple one
- Simple manipulations are still easier with plain text files (just append one to another)
- This case is a pretty failover one so I don't expect anything complicating about this list - as well as possible extending
- For CSV we'll need a new external dependency

Reply:
To make my position easier to understand, I will follow your example with structuring:

Pros for both CSV and JSON:
- A well-known format requires less documentation and users will feel more comfortable about its syntactic rules.
- A typical use case of the alternative initial peers feature includes getting the actual peers from some running node. This part will not likely be done by hand and requires automation. This means that peer lists will most likely be generated by some scripts. These scripts are easier to write and maintain when they use standard formatting tools.

Pros for CSV:
- all the manual manipulation ease you mentioned for our custom format

Cons against CSV:
- the need for new parser code

Pros for JSON:
- more extension capabilities

Cons against JSON:
- harder to process without parsing